src/index.js

```diff
@@ -0,0 +1,80 @@
+// import _ from 'lodash';
+import './style.css';
+import todoTasks from './modules/task';
+import {
+  addTask,
+  deleteTask,
+  addNewTask,
+  toggleEditMode,
+  saveToLocalStorage,
+} from './modules/addremove';
+
+const listDiv = document.querySelector('.list-items');
+const clearBtn = document.querySelector('.clear');
+const inputField = document.querySelector('.task');
+
+const buildTask = () => {
+  listDiv.innerHTML = '';
+  todoTasks.forEach((task) => {
+    addTask(task.description, task.index, task.completed);
+  });
+
+  const checkboxes = document.querySelectorAll('.check-box');
+  checkboxes.forEach((checkbox) => {
+    checkbox.addEventListener('change', () => {
+      const index = Number(checkbox.id.split('-')[0]);
+      const status = todoTasks[index - 1].completed;
+      todoTasks[index - 1].completed = !status;
+      saveToLocalStorage();
+    });
+  });
+  saveToLocalStorage();
+};
+
+inputField.addEventListener('keydown', (event) => {
+  addNewTask(inputField, event);
+});
+
+clearBtn.addEventListener('click', () => {
+  const completedIndexes = todoTasks.reduce((indexes, task, index) => {
+    if (task.completed) indexes.push(index);
+    return indexes;
+  }, []);
+
+  if (completedIndexes.length > 0) {
+    deleteTask(completedIndexes);
+    buildTask();
+  }
+});
```

Comment:
As per the project requirements, to clear all completed tasks, you should be using the `filter()` method to delete all the completed tasks. This approach will be much simpler than the current approach you are using.
![image](https://github.com/albertkantwi/to_do_list/assets/33268927/301e0010-2ff7-4f1e-8390-ab083c594a17)